src/ConfigFactory.php

```diff
@@ -110,6 +110,7 @@ public static function createConfig($localHome = null)
     public static function createConfigFromEnv($systemConfigEnv = null, $localConfigEnv = null)
     {
         $cacheDir = '/tmp';
```

Comment:
I have been removing caching things for a while now, it is an example of a bad decision gone rogue 😊